kafka/client_async.py

```diff
@@ -821,10 +825,10 @@ def check_version(self, node_id=None, timeout=2, strict=False):
             This is only possible if node_id is None.
 
         Returns: version tuple, i.e. (0, 10), (0, 9), (0, 8, 2), ...
+            (0, 0) returned if the version cannot be determined,
+            typically due to networking.
 
         Raises:
-            NodeNotReadyError (if node_id is provided)
-            NoBrokersAvailable (if node_id is None)
```

Comment:
So what happens if bootstrapping eventually fails? Will these still get raised somewhere else in the stack?

I know there's code in the wild that is expecting these exceptions, as I used it myself when I updated the Datadog kafka consumer lag check to use the async client instead of the old SimpleClient. 

To clarify, I'm not against this change, I just want to understand the new behavior.